### ISSUE TITLE
Logo design

### ISSUE THREAD

**Issue by memoryskull** [rashadg1030/game-of-life#4]
Hi!

I'm interested in collaborating with this project by delivering a series of logotype options that fits this project for the best!

If interested let me know so we can get started!

Cheers!

**Comment by rashadg1030** [comment]
Yeah sure! What did you have in mind?

**Comment by memoryskull** [comment]
Hi @rashadg1030 
Could you give me information about your project?
I can submit a proposal without any commitment if it is to your liking I can add it to your repository

**Comment by memoryskull** [comment]
hi @rashadg1030
the name of the game is Conway's Game Life?

**Comment by rashadg1030** [comment]
Yes, that’s correct! Feel free to contribute some artwork. You can find more information about the game here https://en.m.wikipedia.org/wiki/Conway%27s_Game_of_Life
It’s a game with a long history in computer science.

**Comment by memoryskull** [comment]
Hi @rashadg1030 
This is my proposal.
I can make any modification you want, be it color or design in general.
![Game of Life](https://user-images.githubusercontent.com/52359514/60762227-d984b600-a028-11e9-988d-2b9e9bc0629a.png)

**Comment by rashadg1030** [comment]
@memoryskull I like the color and design! I would like one change though. Can you make the squares in the shape of this structure? http://www.conwaylife.com/wiki/R-pentomino
This a popular pattern in the game of life.

**Comment by memoryskull** [comment]
Hi @rashadg1030 
Perfect here I attach the logo with the necessary corrections.
![gameoflifeNEW](https://user-images.githubusercontent.com/52359514/60774904-1617e680-a0e9-11e9-9898-f9a950514874.png)

**Comment by rashadg1030** [comment]
Looks 👍🏽

**Comment by memoryskull** [comment]
@rashadg1030 
I would like to accept the pr for the contribution of the logo.

**Comment by memoryskull** [comment]
@rashadg1030
I would like to accept the pr for the contribution of the logo.
